schema.sql

```diff
@@ -27,3 +27,27 @@ ALTER TABLE animals ADD COLUMN owners_id INT;
 
 ALTER TABLE animals ADD CONSTRAINT fk_owners FOREIGN KEY (owners_id) REFERENCES owners(id);
 
+-- Create the vets table
+CREATE TABLE vets (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(255),
+    age INT,
+    date_of_graduation DATE
+);
+
+-- Create the specializations table
+CREATE TABLE specializations (
+    vet_id INT,
+    specialization VARCHAR(255),
```

Comment:
:red_circle: There is a many-to-many relationship between the tables species and vets.  This "join table"  specializations should contain two relationships one for `vets` and another one for `species`. Currently, It has only one vet_id. The specialization column should be renamed to species_id to make it consistent with the naming convention for primary keys.
```
 CREATE TABLE specializations (
   species_id int,
   vet_id int
 );

```